guidelines.md

```diff
@@ -15,6 +15,14 @@ _there no rule of thumb of exact duration and number of times, and it is part of
   - monitoring and observability _e.g: datadog, apm_
   - Data software and ETL _e.g: Fivetran_
 
+- The `data stack`is composed of:
+  - Database
+  - Mixpanel
+  - Datadog
+  - Fivetran
+  - Metabase
+  - Heroku
+  - Data pipeline (all the steps to fill Metabase's data)
```

Comment:
Nice addition!

One suggestion is to _reverse_ the list like above. Talk first about the concept that important to master, then the tools associated at moka.

For instance
- tracking _e.g: Mixpanel_
- monitoring and observability _e.g datadog, tool dans metabase?_

etc...
----

This does not bind to our current tooling (which might change), and focus more and _what_ you can do vs the _how you do it_. IMHO, what's important is to know how to look and build dashboard to monitor the DB (what's interesting to monitor, how to collect basic metrics, etc) for instance, be it in datadog, or any other tools